cmd/activator/main.go

```diff
@@ -63,12 +62,9 @@ import (
 var _ = goversion.IsSupported()
 
 const (
-	maxUploadBytes = 32e6 // 32MB - same as app engine
-	component      = "activator"
+	component = "activator"
 
-	maxRetries             = 18 // the sum of all retries would add up to 1 minute
-	minRetryInterval       = 100 * time.Millisecond
-	exponentialBackoffBase = 1.3
+	maxRetries = 18 // the sum of all retries would add up to 1 minute
```

Comment:
I think we agreed this 1 minute is no longer correct anywhere? :-)

Reply:
Done.